picard/options.py

```diff
@@ -208,6 +208,11 @@
 BoolOption('setting', 'save_images_to_files', False, title=N_("Save cover images as separate files"))
 BoolOption('setting', 'save_images_to_tags', True, title=N_("Embed cover images into tags"))
 BoolOption('setting', 'save_only_one_front_image', False, title=N_("Save only a single front image as separate file"))
+BoolOption('setting', 'show_cover_art_details', False, title=N_("Show cover art details in cover art view"))
+BoolOption('setting', 'show_cover_art_details_type', True, title=N_("Show cover art type in info labels"))
```

Comment:
I would default this specific setting to `False`, as the cover art box would usually always show the front cover. But no strong opinion, just my view on what is the better default.